docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -627,9 +596,78 @@ simplifying assumptions that we have included:
 can use an interactive simulation in a Jupyter Notebook to check that
 simulants are moving at the expected rates.
 
+.. _census_prl_international_immigration:
 
-2.3.5 Component 9: Address
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+2.3.5 Component 9: International Immigration (In-Migration)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+New simulants are added by migration into the US from other countries.
+We simulate two kinds of immigration: household moves and non-reference-person moves.
+
+#. A **household move** is when an entire household (which may be a single person) enters from outside the country as a unit,
+   preserving relationships within the unit.
+#. A **non-reference-person move** is when a person enters from outside the country and joins an existing household.
+   Non-reference-person moves are independent, single-person events that do not preserve relationship structure.
+
+For the purposes of immigration, a group quarters person should be treated identically to a one-person household,
+with the group quarters person as the reference person.
+This means that group quarters people should only enter the US in a household move.
+
+The number of simulants who move to the US each year is informed by the ACS' "residence one year ago" question.
+Specifically, a value of 2 for variable :code:`MIG` indicates that a respondent lived outside the US one year ago,
+while any other value indicates that they lived within the US.
+We refer to respondents who were not living in the United States one year ago as "recent immigrants."
+Our assumption is that the number (and characteristics) of recent immigrants
+in the 2019 ACS will be replicated in each future year.
+
+We also assume that the proportions of recent immigrants by move type (household or non-reference-person) will remain constant.
+Though in reality not all moves into the US follow one of these two patterns, we assume that any new immigrant in a household
+where the reference person is also a new immigrant was part of a household move, while any new immigrant in a household where the
+reference person is not a new immigrant moved to the US in a non-reference-person move.
+
+Specifically, the yearly rate at which simulants are added to the population by each move type is given by
+the (weighted) proportion of ACS PUMS persons that are recent immigrants consistent with that move type.
+At each time step:
+
+#. PUMS households with reference people who are recent immigrants, after removing household members who are not recent immigrants,
```

Comment:
i assume this is per-state? i.e., we look sample from all 2019 **florida** residents who are recent immigrants?

Reply:
also, are we sampling with the household weight or the person weight of the household head, here? on initialization we use the household weight to sample households, so i assume we'd do that again here -- but that may(?) create strange behavior in the case that there is some non-reference person with a high person weight, who is a recent immigrant, living in a household with a recent immigrant household head, with a low household weight. << not sure if that scenario exists in the data, but this would result in that non-reference person being unlikely to enter the simulation, despite having a high person weight

this is almost certainly not a concern, but wanted to note it might contrast with the way we sample the non-reference recent immigrants living in houses with non-recent immigrant household heads

Reply:
As currently written, when we sample recent immigrants, we preserve their PUMA and state. So the sampling happens at the national level, but the immigrants within each state (and indeed PUMA) will all be based on recent immigrants to that state/PUMA.

Re: weights, this was an oversight on my part. I believe it will be correct to sample households using household weights and individuals using person weights, even though it is a bit odd. I will clarify this in the documentation.

Reply:
Ah, I think I may have misunderstood what you meant by your first question. When we are only _simulating_ Florida and not the entire US, all ACS data used should be filtered to Florida.